infogami/infobase/_dbstore/save.py

```diff
@@ -386,7 +386,7 @@ def get_pid(type, name):
 
         dbindex = {}
 
-        for (type, key, datatype, name), values in index.iteritems():
+        for (type, key, datatype, name), values in index.items():
```

Comment:
Used only once.  Python 2 will build a list in memory which will take up more RAM.  Python 3 will be fast.